src/routes/login/index.js

```diff
@@ -4,12 +4,15 @@ import express from 'express'
 export default function () {
   const router = express.Router()
 
+  var showGoogleSSO = false
+  if ( process.env.GOOGLE_CLIENT_ID && process.env.GOOGLE_CLIENT_SECRET ){ showGoogleSSO = true }
```

Comment:
This should be moved to src/config/index.ts if possible.

Reply:
Fair. I've been able to remove this by recycling some existing variables: https://github.com/Wingysam/Christmas-Community/pull/138/commits/5d58cb5e3531ef5fc36618ad65429ab8612c0026.